tests/cypress/cypress/pages/Event.js

```diff
@@ -0,0 +1,208 @@
+import 'cypress-iframe'
+import content from './Content'
+import testData from './TestData'
+class Event {
+
+    //Place page through mouse hover on admin tool bar
+    get eventLink() {
+        return cy.get("#toolbar-item-administration-tray > nav > div.toolbar-menu-administration > ul > li:nth-child(2) > ul > li:nth-child(2) > ul > li:nth-child(2) > a")
+    }
+    //get page title
+    get eventEditTitle() {
+        return cy.get("#edit-title-0-value")
+    }
+    //get HTML edit box
+    get eventBodyEdit() {
+        return cy.get("#edit-body-wrapper > div > div.js-form-type-textarea.js-form-item.form-item.form-type--textarea.js-form-item-body-0-value.form-item--body-0-value > div")
+    }
+    //get the format of editing
+    get eventTextFormat() {
+        return cy.get("#edit-body-0-format > div.form-item--editor-format.js-form-item.form-item.js-form-type-select.form-type--select.js-form-item-body-0-format.form-item--body-0-format > label")
+    }
+    //get the format of editing, dropdown
+    get eventTextFormatDropdown() {
+        return cy.get("#edit-body-0-format--2")
+    }
+
+    //get start and end date/time
+    get startDate() {
+        return cy.get("#edit-field-event-start-0-value-date")
+    }
+
+    get startTime() {
+        return cy.get("#edit-field-event-start-0-value-time")
+    }
+
+    get endDate() {
+        return cy.get("#edit-field-event-end-0-value-date")
+    }
+
+    get endTime() {
+        return cy.get("#edit-field-event-end-0-value-time")
+    }
+    //get door date
+    get doorDate() {
+        return cy.get("#edit-field-door-time-0-value-date")
+    }
+    get doorTime() {
+        return cy.get("#edit-field-door-time-0-value-time")
+    }
+
+    //get event duration
+    get eventDuration() {
+        return cy.get("#edit-field-event-duration-0-value")
+    }
+    //get place of event
+    get eventPlace() {
+        return cy.get("#edit-field-event-place")
+    }
+    //get tags of the event
+    get eventTags() {
+        return cy.get("#edit-field-tags-target-id")
+    }
+    //get type of the event
+    get eventType() {
+        return cy.get("#edit-field-event-type")
+    }
+    //get language for the event
+    get eventLanguageSelect() {
+        return cy.get("#edit-langcode-0-value")
+    }
+    //Save as dropdown
+    get eventSaveAsDropdown() {
+        return cy.get("#edit-moderation-state-0-state")
+    }
+    //save the event button
+    get eventSave() {
+        return cy.get("#edit-submit")
+    }
+
+    //Validation
+    //Fetch created event title
+    get fetchCreatedTitle() {
+        return cy.get("body > div.dialog-off-canvas-main-canvas > div.coh-container.coh-style-focusable-content.coh-ce-6f78460f > div > article > div.coh-container.coh-style-padding-top-medium.coh-style-padding-bottom-large.coh-container-boxed > div:nth-child(1) > div > div > h1")
```

Comment:
Common elements: body > div.dialog-off-canvas-main-canvas > div.coh-container.coh-style-focusable-content.coh-ce-6f78460f > div > article > div.coh-container.coh-style-padding-top-medium.coh-style-padding-bottom-large.coh-container-boxed > div:nth-child(1) > div > div

In fetchCreatedTitle and fetchPlaceEvent